targets/targets.json

```diff
@@ -668,367 +668,321 @@
         "extra_labels_remove": ["FRDM"],
         "supported_form_factors": []
     },
+    "FAMILY_STM32": {
+        "inherits": ["Target"],
+        "core": "",
```

Comment:
It's required for public targets, but you don't really want to be able to build for the non-target "FAMILY_STM32".